modules/nf-core/abacas/tests/main.nf.test

```diff
@@ -0,0 +1,38 @@
+nextflow_process {
+
+    name "Test Process ABACAS"
+    script "../main.nf"
+    process "ABACAS"
+    config "./nextflow.config"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "abacas"
+
+    test("sarscov2 [scaffolds_fasta] [genome_fasta]") {
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test', single_end:false ], // meta map
+                             file(params.test_data['sarscov2']['illumina']['scaffolds_fasta'], checkIfExists: true)
+                           ]
+                input[1] = file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.results[0][1].findAll {
```

Comment:
I think we should check at least the filename for the files excluded here.

Reply:
good point...don't think its done implicitly....will add it in...give me few min

Reply:
plz check and merge if ready. thanks